mailgreen/services/carbon_service.py

```
@@ -0,0 +1,110 @@
+        co2 = 75.0  # 대형 첨부 (75g)
+    return co2 * 0.0025  # kWh로 환산
+
+
+def get_carbon_stats_service(user_id: str):
+    db = SessionLocal()
+    try:
+        now = datetime.utcnow()
+        # 1. 삭제된 메일 전체 조회
+        deleted_mails = db.query(MailEmbedding).filter(
+            MailEmbedding.user_id == user_id,
+            MailEmbedding.is_deleted == True,
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if MailEmbedding.is_deleted:` for truth checks

Replace with `MailEmbedding.is_deleted`

(E712)